### PR TITLE
Land app-elements

### PR DESCRIPTION
**Pull request by blasten**
There are a few issues in the app-elements that need to be addressed before merging this PR. cc @tjsavage 

UI issues:

* [x] The prefix `app` barely fits the card.
* [x] The main navigation for an element doesn't work very well when there are too many demos.

### CODE REVIEW

**Comment by frankiefu** [comment]
- The abbreviation could be 1 letter ("A") or 2 letters (e.g. "Ae").  But that involves more discussions.  Maybe for now just keep "App" and make the font-size smaller, maybe 16px.

- We definitely need to redesign the Catalog to better support multiple elements in a single repo.  But for now maybe just add max-height (relative to viewport like vh) to the top section if there are bundled elements in it.  Make sure it works ok on small screen (like mobile).

**Comment by arthurevans** [comment]
+1 for keeping App.

File: app/elements/app-sidebar/app-sidebar.css [hunk 1]
```diff
@@ -48,5 +48,7 @@
 }
 
 ::content .content {
-  @apply(--layout-vertical);
+  overflow-x: hidden;
+  overflow-y: auto;
```
Comment (frankiefu) [inline app/elements/app-sidebar/app-sidebar.css]: add `-webkit-overflow-scrolling: touch;`

**Comment by frankiefu** [comment]
LGTM

**Pull request merged**